discovery/kubernetes/service.go

```diff
@@ -233,7 +234,21 @@ func (s *Service) buildService(svc *apiv1.Service) *targetgroup.Group {
 		}
 
 		if svc.Spec.Type == apiv1.ServiceTypeLoadBalancer {
-			labelSet[serviceLoadBalancerIP] = lv(svc.Spec.LoadBalancerIP)
+			if len(svc.Status.LoadBalancer.Ingress) > 0 {
+				ips := make([]string, 0, len(svc.Status.LoadBalancer.Ingress))
+				for _, ing := range svc.Status.LoadBalancer.Ingress {
+					if ing.IP != "" {
+						ips = append(ips, ing.IP)
+					}
+				}
+				if len(ips) > 0 {
+					labelSet[serviceLoadBalancerIP] = lv(strings.Join(ips, ","))
```

Comment:
`__meta_kubernetes_service_loadbalancer_ip` is only meant to contain one IP address https://prometheus.io/docs/prometheus/latest/configuration/configuration/#service
let's me think about this.